common/config/rush/common-versions.json

```diff
@@ -16,12 +16,8 @@
      * instead of the latest version.
      */
     // "some-library": "1.2.3"
-
-    // This is required to allow for backward compatibility with Service Bus Track 1
-    // Workaround for microsoft/web-build-tools#1415
-    // TODO: Remove this once Service Bus is updated to use current depenedencies as part of Track 2
-    "rhea-promise": "^0.1.15"
   },
+  "implicitlyPreferredVersions": false,
```

Comment:
Could you leave a comment here to explain what this does?  The name of the field isn't very self-explanatory

Reply:
I added the comment generated by running `rush init` in an empty folder.  I will update all our Rush config files with the latest comments generated by `rush init` in a separate PR.